WordPress/Classes/ViewRelated/Notifications/Milestone Notifications/ConfettiView.swift

```diff
@@ -177,23 +178,36 @@ extension ConfettiView {
         }
 
         // Colors
-        let purple = UIColor(red: 0.75, green: 0.35, blue: 0.95, alpha: 1.00)
-        let orange = UIColor(red: 1.00, green: 0.50, blue: 0.52, alpha: 1.00)
-        let green = UIColor(red: 0.44, green: 0.88, blue: 0.65, alpha: 1.00)
+        let purple = UIColor(light: UIColor(named: "Purple20") ?? .systemPurple,
```

Comment:
You can use our Muriel colors here – check `MurielColor.swift` and `UIColor+MurielColors.swift`. I would suggest adding this convenience method to the top of `UIColor+MurielColors`:

```
class func muriel(name: MurielColorName, _ shade: MurielColorShade) -> UIColor {
    let newColor = MurielColor(name: name, shade: shade)
    return muriel(color: newColor)
}
```

and then you'll be able to reference these colours like: `.muriel(name: .purple, .shade20)`

Reply:
Done, thanks for the suggestion!